qp_woltka/tests/test_woltka.py

```diff
@@ -138,6 +133,7 @@ def test_woltka_to_array_rep82(self):
         with open(merge_fp) as f:
             merge = f.readlines()
 
+        prep_file = f'{out_dir}/prep_info.tsv'
```

Comment:
I'm not a stickler for using os.path.join() over using '/' but I believe most or all of the qiita code uses the former.

Reply:
changing all